### DISCUSSION TITLE
Unknown Exchange

### DISCUSSION THREAD

**Discussion by Erik5000** [starsoccer/cryptotithe#293] [Q&A]
hey

If I try to load the trades.csv from Kraken I get this in my console:
Uncaught (in promise) Error: Unknown Exchange -  - acf4523af1f344d5edd6887a6feeffa8d6d6096c015661f0b4efe78d3d18d4e6
    at O (import-5690300b6c393c3b.js:1:214660)
    at V (import-5690300b6c393c3b.js:1:216373)
    at Object.onLoaded (import-5690300b6c393c3b.js:1:231307)
    at e.onload (index-7519b7a6a169dcbb.js:1:811)

If go for recalculate holdings, it's just spitting out at data.json in my download folder.
I am not sure how to use this. Please help :)

**Comment by starsoccer** [comment]
This is likely just because the hash for kraken has changed recently. For now if you simply select trades and then choose kraken from the drop down, it should import fine. If not let me know.
